connector/oidc/oidc.go

```diff
@@ -265,6 +275,12 @@ func (c *oidcConnector) LoginURL(s connector.Scopes, callbackURL, state string)
 	if s.OfflineAccess {
 		opts = append(opts, oauth2.AccessTypeOffline, oauth2.SetAuthURLParam("prompt", c.promptType))
 	}
+
+	if c.pkceVerifier != "" {
+		c.pkceVerifier = oauth2.GenerateVerifier()
```

Comment:
Isn't this setting just one verifier for all clients?  It appears that if 2 clients were logging in at the same time the first one's verifier would be overwritten by the seconds preventing a proper verifier exchange (line 317-320).